components/JCSwitch/JCSwitch.tsx

```diff
@@ -57,7 +57,7 @@ export default class JCSwitch extends JCComponent<Props, State> {
       duration: 200,
       useNativeDriver: true
     }).start();
-    this.setState({ enabled: !this.state.enabled }, () => { this.props.onPress(this.state.enabled) }
+    this.setState({ enabled: !this.state.enabled }, () => { this.props.onPress() }
```

Comment:
I added this because of changes needed when working with a db...

https://github.com/jesus-collective/mobile/commit/e198dceaa5ab5844da87d57c601529b638b81f73 - the value could get flipped and be out of sync with the DB....you can play with it a bit by removing that isEditable if statement around the switch on the event or group page...so you can see the Sponsored text and see what happens...

Reply:
Ad ok - I saw what happens. Reverted in the latest commit.